admin_guide/manage_rbac.adoc

```diff
@@ -762,8 +762,158 @@ To bind the new role to a user, run:
 $ oc adm policy add-role-to-user podview user2 --role-namespace=bob-project -n bob-project
 ----
 
+[[creating-cluster-role]]
+== Creating a new cluster role
+
+To create a cluster role, run the following command:
+
+----
+$ oc create clusterrole -h ...
+----
+
+The following excerpt from the help of this command describes its usage:
```

Comment:
This is unusual for our docs, I think. I see it used elsewhere in this topic, but it feels like bad form. Can these be bulleted instead and remove the "excerpt" intro?

Reply:
@mburke5678 @bfallonf, I'll tidy up both sections. (It is a slippery slope, and I originally had only the next new section.) Thank you!